src/boot/stylesheets/spinner.scss

```diff
@@ -1,138 +1,40 @@
-//CSS Spinner http://codepen.io/MichaelArestad/pen/4f837dd51dc09256115afc9218edda2c
-@keyframes ui-wpnc__spinner-rotate-right {
-  0% {
-    transform: rotate(0deg);
+.wpnc__main {
+  .wpnc__spinner {
+    position: relative;
+    top: 50%;
   }
-  25% {
-    transform: rotate(180deg);
-  }
-  50% {
-    transform: rotate(180deg);
-  }
-  75% {
-    transform: rotate(360deg);
-  }
-  100% {
-    transform: rotate(360deg);
-  }
-}
-
-@keyframes ui-wpnc__spinner-rotate-left {
-  0% {
-    transform: rotate(0deg);
-  }
-  25% {
-    transform: rotate(0deg);
-  }
-  50% {
-    transform: rotate(180deg);
-  }
-  75% {
-    transform: rotate(180deg);
-  }
-  100% {
-    transform: rotate(360deg);
-  }
-}
-
-$size: 20px;
-$transition-speed: .4s;
-.wpnc__spinner {
-  position: relative;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  border-radius: 100%;
-  width: $size;
-  height: $size;
-  background: lighten($gray, 20);
-
-  // before & after make the donut hole
-  &:before, &:after {
-    content: '';
-    position: absolute;
-    background: $white;
-    border-radius: 50%;
-  }
-  &:before {
-    $stroke: 1px;
 
-    width: $size - ($stroke * 2);
-    height: $size - ($stroke * 2);
-    top: $stroke;
-    left: $stroke;
-  }
-  &:after {
-    $stroke: 3px;
-
-    width: $size - ($stroke * 2);
-    height: $size - ($stroke * 2);
-    top: $stroke;
-    left: $stroke;
-  }
-  &:hover {
-    .left:before {
-      transition-delay: $transition-speed;
-    }
-    .right:before {
-      transition-delay: 0;
-    }
-  }
-  .side {
-    width: 50%;
-    height: 100%;
-    overflow: hidden;
-    position: absolute;
-
-    &:before {
-      content: '';
-      border-radius: 500px;
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      background: $blue-medium;
-      transition: all $transition-speed .4s linear;
+  // Start https://github.com/Automattic/wp-calypso/blob/master/client/components/spinner/style.scss
+  @keyframes rotate-spinner {
+    100% {
+      transform: rotate( 360deg );
     }
   }
-  .left {
-    left: 0;
 
-    &:before {
-      left: 100%;
-      border-top-left-radius: 0;
-      border-bottom-left-radius: 0;
-      transform-origin: 0 50%;
-      transition-delay: 0;
-    }
+  .spinner {
```

Comment:
Out of curiosity is there a reason why we don't namespace all of our classes? Or did we feel the parent class was enough?

Reply:
when it was young we were in our own iframe and the app was small and we had no felt need to namespace.

Reply:
@dmsnell would we like to namespace the rest? I feel like that's usually the safest

Reply:
yeah that's great. we started using `wpnc_` as the prefix. no reason not to add it everywhere else. I predict that we only put it where we needed to in order to integrate this code directly into Calypso